docker-compose.yml

```diff
@@ -1,5 +1,3 @@
-version: '3'
```

Comment:
What's up with the compose version removal?

Reply:
Unrelated change but docker-compose complained about the version being deprecated every time, so I've finally removed it.